hotham/src/systems/grabbing.rs

```diff
@@ -69,10 +73,13 @@ mod tests {
 
         let grabbed_collider = ColliderBuilder::cuboid(1.0, 1.0, 1.0).build();
         let grabbed_rigid_body = RigidBodyBuilder::new_dynamic().build();
-        let grabbed_entity = world.spawn((Info {
-            name: "Test entity".to_string(),
-            node_id: 0,
-        },));
+        let grabbed_entity = world.spawn((
```

Comment:
Nice! Can you write a test confirming the inverse? (eg. objects without grabbable can't be grabbed)?